### PR TITLE
New rip page

### PR DESCRIPTION
**Pull request by JJGothard**
@claireriley , give this a look over when you can. There were some snafus, like markdown preview not being available for new docs in Atom. Let me know how it all came out on the other side :) 

Thanks!

### CODE REVIEW

**Comment by JJGothard** [comment]
@claireriley The commits may be kind of scattered because I renamed the branch mid-project, and that wasn't reflected in the repo. As a result, all of the commits were visible to me in the app, but there were one or two missing from the commit history for the original branch name in the repo. I ended up just copying the entire contents, deleting them, pushing that commit; and then pasting them back into the file, and committing and pushing that change. So if it's possible to review the final commit only, that should encompass all of the work. 

Thanks!

**Comment by JJGothard** [comment]
Actually, @claireriley , now that I'm trying to go through it, it looks like none of that has worked. You know more about reviewing commits, so I'll wait to hear from you.

**Comment by SueSmith** [comment]
@JJGothard could you fix the filename issue as discussed then run this locally, give it another once over once you can see how it all renders in the site, then ping me for a review? 🙂

**Comment by JJGothard** [comment]
Doing now @SueSmith, thanks!

**Comment by JJGothard** [comment]
@SueSmith , Okie doke, I think I've reached the point of diminishing returns on tweaking every sentence now. Need feedback, ready for review, I think. Thank you!

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Introduction to the Run in Postman button"
+order: 332
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: I'd put this one before the create one so it comes up first in the folder - we also have two with the same number now (332), you can use decimal places here so e.g. 330.9 would put it first in the dir without having to change any others

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+page_id: "introduction_run_button"
+warning: false
+
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: can we add the sidebar links here (see other pages in the folder for examples)

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+The Run in Postman button is a way to give users working with your APIs direct access to your endpoints. When you [create a button](/docs/postman-for-publishers/run-in-postman/creating-run-button/) for a collection and then embed it in your API documentation, website, or GitHub repo, a copy of that collection will open in the Postman app whenever a user clicks the button.
+
+![Travefy button](https://assets.postman.com/postman-docs/Travefy+API+RIP+button.jpg)
+
+> The Run in Postman button works with all description formats like Swagger, RAML, and GitHub.
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: maybe lose "all" just to be on the safe side here lol

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## Button types & including an environment
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: can we reword this heading? would lose "&" in general and maybe make more specific if possible

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+There are 2 types of Run in Postman buttons:
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: let's spell out words up to ten, and make everything before a colon a complete sentence

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+**Dynamic button**: Uses HTML and CSS. Optimal for a website where you have permission to edit the site's source code. Gives you access to the [Run in Postman API](/docs/postman-for-publishers/run-in-postman/run-button-API/).
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: isn't this on another page? let's not repeat content in more than one place if we can avoid it

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+**Static button**: Uses Markdown. Optimal for a Markdown document like a GitHub ReadMe or BitBucket repository.
+
+With both buttons you have the option to [include an environment](/docs/postman-for-publishers/run-in-postman/environments-run-button/). The environment will open in the Postman app along with a copy of your collection whenever a user clicks your button. The user can then input their own [variable values in the environment](/docs/postman/variables-and-environments/variables/).
+
+## Access to the Run in Postman API
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: maybe "accessing the.."?

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+The JavaScript [Run in Postman API](/docs/postman-for-publishers/run-in-postman/run-button-API/) is accessible through the dynamic Run in Postman button. The API uses the `pm()`method, which you can leverage to create and modify environments included in your button. You can also include multiple buttons on a single page with segregated environments, each environment with its own button.
+
+The API consists of 3 JavaScript functions that can alter your button's behavior:
+
+* `env.create`: Creates a new environment.
+* `env.assign`: Edits an existing environment.
+* `env.replace`: Replaces and entire environment.
+
+## User interaction with your button
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: maybe reword to be more active?

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+When a user finds your button displayed on your website or GitHub ReadMe and they click it, they'll be prompted to open a copy of the collection in the Postman app. If they haven't downloaded the app yet, they'll have the option to do so then.
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: I'd lose "github readme"

Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: "they'll be prompted to open the Postman app"

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+ ![Modal display](https://assets.postman.com/postman-docs/New+sized+Modal+RIP+button.jpg)
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: screengrab is very pixelated, can we get a better quality one? would also mention this is specifically the mac one, or just remove it so it isn't confusing to non mac users

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+
+> If you're using the dynamic button, the prompt will display in a modal. If you're using the static button, the prompt will open in a new page.
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: not sure I'm following this - don't think the behavior is different for static and dynamic from the consumer point of view..?

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+
+The app will open on their desktop and automatically have your collection opened and ready for use.
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: would clarify "their"

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+
+## Security considerations
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: how does this section relate to the security considerations page? might be better to redraft the content on there instead of having a section here as well

**Comment by JJGothard** [comment]
@SueSmith , yes, currently there is repetition. Once this is approved, I'll deprecate the other pages that have been repeated here, and this will be where all of that info lives. 

Will go through your comments shortly

**Comment by SueSmith** [comment]
@JJGothard thanks for clarifying, hm it's going to be difficult to get that right without us seeing all of the changes in one pr, if you're consolidating info let's try and do it all in the same place so we can get an overview of the whole picture. 👍 Bearing in mind that a pr is merged and published as a unit so the other info would still be in the site when the repeated info is added. I know we talked about using branches for single enhancements but I'd describe that as a single piece of work if that makes sense.. 🙂

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Introduction to the Run in Postman button"
```
[the remark this answers is not in the document]
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Done

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 1 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: done

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 2 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: done

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 3 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: done

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 4 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: broke up into 1 heading + 1 subhead to be more specific but still scannable

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 5 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: done

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 6 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: It's in the creating page button, but I think we shouldn't assume that people read docs as entire corpus (in my experience, that's not how users interact with docs). They don't read everything in the LC, they don't read things in order, and they don't necessarily even read everything on a given page. Some content will need to repeat, for that reason.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 7 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Done

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 4 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Reverted back because I don't like the way it looks on the page. Changed wording of header, tho it's long.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 8 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Tweaked a bit. Given that this is a conceptual doc rather than an instructive doc, I think active voice is less of a concern.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 9 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: done

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 10 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Good point about the mac users! Added new screenshot and changed note.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 11 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: It is, I checked. If you're a user, and you click the button the button in someone's docs on their website vs. a markdown button, the prompts (telling the user to open the Postman app or download it) open differently. The latter in a modal, the former in a new page. Reworded to hopefully make that clear -- let me know.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 12 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Done

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 13 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Going to deprecate the other page once this is merged.

**Comment by JJGothard** [comment]
@SueSmith , all new comments have been addressed. Thanks!

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
+---
+title: "The Run in Postman button"
+order: 330.9
+page_id: "introduction_run_button"
+warning: false
+contextual_links:
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: can we structure these in sections as in the other pages?

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+---
+
+The Run in Postman button is a way to give users working with your APIs direct access to your endpoints. When you [create a button](/docs/postman-for-publishers/run-in-postman/creating-run-button/) for a collection and then embed it in your API documentation, website, or GitHub repo, a copy of that collection will open in the Postman app whenever a user clicks the button.
+
+![Travefy button](https://assets.postman.com/postman-docs/Travefy+API+RIP+button.jpg)
+
+> The Run in Postman button works with description formats like Swagger, RAML, and GitHub.
+
+## Dynamic & Static buttons with environments
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: can we do "and" instead of "&"

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 6 above)
Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: that's why we use modular content, so that we can link out to supporting info to provide what people need to understand without repeating info (which creates multiple points of change) - we wouldn't repeat the same info in more than one place unless it's unavoidable

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+
+* `env.create`: Creates a new environment.
+* `env.assign`: Edits an existing environment.
+* `env.replace`: Replaces and entire environment.
+
+## User interaction with your button
+
+When a user finds your button displayed on your website they click it, they'll be prompted to open the Postman app. If they haven't downloaded the app yet, they'll have the option to do so then.
+
+ ![Modal display](https://assets.postman.com/postman-docs/Modal+new+size+quality.jpg)
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: screengrab still looks unclear

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 11 above)
Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: ahh I'm with you, it pops up above the current page vs a second page, thanks for clarifying

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+
+> If the button is dynamic, the prompts will open for the user in a modal, like the screenshot above. (The modal above is specific to Macs.) If the button is static, the prompts will open for the user in a new page.
+
+The app will open on the user's desktop and automatically have your collection opened and ready for use.
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: I don't believe the collection opens, it just imports right?

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 13 above)
Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: so we would need to do that as part of the same pr, merging is publishing, so if something is being consolidated it needs to happen in one go

Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Got it. I'll go through deprecation process today.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 6 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: I'll remove from the "Creating the Run in Postman button" and replace with a link to this doc. I'd need to wait for this doc to be published, yes?

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 15 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Sure, but I'm not 100% sure what you mean by that. I used the "Sharing collections" doc as a model, and I see some of the contextual link types called sections, but they don't seem to directly correlate to the sections in the doc or the sections in the doc/place that they're linked to.

**Comment by JJGothard** [comment]
@SueSmith , I'm deleting the already discussed files for this PR, but I've never added a "redirect" before. Can you instruct me here or ping me on Slack? Thanks!

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 6 above)
Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: if you have edits that are dependent on other edits let's do them in the same pr, as long as you use relative links we can see the whole picture at the same time that way

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 15 above)
Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: have a look at how the links are structured in other pages and use whatever sections make sense for this one, copying the structure from the md in other pages

**Comment by SueSmith** [comment]
@JJGothard so you'll need to add it in the redirects.json file, and search for links to it throughout the docs and update them (I'd recommend using git grep to find the links) - see the breaking urls section in the internal tech writer onboarding doc

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 17 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Just saw this, will try again now.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 6 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Okay. What would be the best way to accomplish that? The whole doc has been rewritten and is in review on another branch. Should I copy the new content into the old file that's on this branch, or is there a better "git" way of handling this? @SueSmith

Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: ^^ The last comment that needs to be addressed for this PR

Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: you could merge one branch into the other using a pull request if you prefer.. also this branch has conflicts that'll need resolved, if you update from develop branch you should be prompted to choose whether to accept the changes on your current branch or the updates from remote (this can get messy btw 😬)

Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: if you end up finding it easier to just copy changes from one branch into the other I'd just go for it, we can still reference the comments from the other review if we need to..

Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:41]: The repetitive info has been removed and will only live here. Done.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 17 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:61]: Appeared to be the size that was the issue. This one is much clearer. Done

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 18 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:65]: You're right, and this is fixed. Done.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 13 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:67]: All of these files have been deprecated as part of this PR

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 15 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:6]: Done

**Comment by JJGothard** [comment]
@SueSmith , all comments have been addressed. I thought I saw a mssg that there conflicts on this branch that needed to be resolved, but I'm not seeing anything about that either in this PR or when I look at the branch in the repo. Did I imagine that? Thanks!

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 15 above)
Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:6]: This is what I'm seeing in the sidebar - can you try again looking at the sections in other pages and restructuring to emulate it making sure you check how it's rendering in the site 
Screenshot 2020-04-02 at 13 47 39

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+
+The API consists of a series of JavaScript functions that can alter your button's behavior:
+
+* `env.create`: Creates a new environment.
+* `env.assign`: Edits an existing environment.
+* `env.replace`: Replaces and entire environment.
+
+## User interaction with your button
+
+When a user finds your button displayed on your website they click it, they'll be prompted to open the Postman app. If they haven't downloaded the app yet, they'll have the option to do so then.
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:73]: first sentence looking a bit off here, maybe missing a word?

**Comment by SueSmith** [comment]
@JJGothard so am seeing three deleted files but only one redirect, we'll need redirects and link updates throughout docs for any deleted files. And yeah there's a conflict showing in redirects.json that'll need resolved, if you look at the pr here on github you should see it.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md [hunk 20]
```diff
@@ -0,0 +1,92 @@
...
+
+ ![Modal display](https://assets.postman.com/postman-docs/Unsized+modal+image.jpg)
+
+> If the button is dynamic, the prompts will open for the user in a modal, like the screenshot above. (The modal above is specific to Macs.) If the button is static, the prompts will open for the user in a new page.
+
+A copy of your collection will be imported into the user's app, and the app will open on the user's desktop.
+
+## Security considerations
+
+This section only covers security details related to the Run in Postman feature. Refer to our [complete security practices](https://www.postman.com/security) for general details.
```
Comment (SueSmith) [inline src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:83]: let's lose "our" here as per style guide

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 13 above)
Comment (SueSmith) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:81]: is the info from the collections-run-button file included here somewhere? apologies if I'm missing it, losing track of what's where..

**Comment by SueSmith** [comment]
@JJGothard wee status update here?

**Comment by JJGothard** [comment]
Hi @SueSmith ! I'll have all open PRs fully addressed by EOD today. Thanks!

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 15 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:6]: Fixed.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 16 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Done, but I will say that I've gotten conflicting feedback, so it might be a good idea to discuss as a team.

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 13 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md:66]: That was included in my rewrite of "Creating a Run in Postman button", PR[ #2251. ](https://github.com/postmanlabs/postman-docs/pull/2251).

File: src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md (hunk 20 above)
Comment (JJGothard) [reply src/pages/docs/postman-for-publishers/run-in-postman/introduction-run-button.md]: Done

**Comment by JJGothard** [comment]
@SueSmith , all comments have been addressed.

File: package-lock.json [hunk 21]
```diff
@@ -28083,8 +28083,7 @@
         "ansi-regex": {
           "version": "4.1.0",
           "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-4.1.0.tgz",
-          "integrity": "sha512-1apePfXM1UOSqw0o9IiFAovVz9M5S1Dg+4TrDwfMewQ6p/rmMueb7tWZjQ1rx4Loy1ArBggoqGpfqqdI4rondg==",
-          "optional": true
+          "integrity": "sha512-1apePfXM1UOSqw0o9IiFAovVz9M5S1Dg+4TrDwfMewQ6p/rmMueb7tWZjQ1rx4Loy1ArBggoqGpfqqdI4rondg=="
```
Comment (SueSmith) [inline package-lock.json:28086]: shouldn't be any changes to this file in the pr so these changes will need removed from the pr

Comment (SueSmith) [reply package-lock.json:28086]: I'm actually not able to run the site locally when I pull from this branch so will need this sorted before I can go back through the comments.. 😬
